src/live-preview.ts

```diff
@@ -333,9 +333,9 @@ export default class LivePreview {
 
     // Request parent for data sync when document loads
     private requestDataSync() {
-        this.handleUserChange({
-            live_preview: "init", // this is the hash of the live previewd
-        });
+        // this.handleUserChange({
```

Comment:
Do not keep dead code. Remove it and add a commit message describing why

Reply:
done